modules/sandboxed-containers-enable-debug-crio.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/troubleshooting-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-enable-debug-logs_{context}"]
+= Enabling debug logs for {sandboxed-containers-first}
+
+As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
+
+.Procedure
+
+To change the `log_level` in CRI-O:
+
+. Create a YAML file for the `ContainerRuntimeConfig` CR with the following manifest:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: crio-debug
+spec:
+ machineConfigPoolSelector:
+  matchLabels:
+    pools.operator.machineconfiguration.openshift.io/worker: '' <1>
+ containerRuntimeConfig:
+    logLevel: debug
+----
+<1> Specify a label for the machine config pool that you want you want to modify.
+
+. Create the `ContainerRuntimeConfig` CR.
++
+[source,terminal]
+----
+$ oc create -f ctrcfg.yaml
+----
+
++
+[NOTE]
+====
+The file name listed above is a suggestion. You can create this files using another name.
+====
+
+. Verify that the CR is created.
++
+[source,terminal]
+----
+$ oc get ctrcfg
+----
++
+.Example output
+[source,terminal]
+----
+NAME           AGE
+overlay-size   3m19s
```

Comment:
I believe it's better to give the actual output in our case as example, hence  **crio-debug** instead of **overlay-size**